pystarport/cluster.py

```
@@ -1445,6 +1445,11 @@ def supervisord_ini(cmd, validators, chain_id, start_flags="", cmd_flags=""):
 
 def supervisord_ini_group(chain_ids, is_hermes):
     directory = "%(here)s"
+    data_dir_hash = hashlib.md5(
```

Ruff (0.14.4)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)